codegen-luau/runtime/runtime.lua

```diff
@@ -327,6 +327,10 @@ do
 		end
 	end
 
+	function trunc.num(num)
+		return if num < 0 then math.ceil(num) else math.floor(num)
```

Comment:
Should also utilize `math_ceil` and `math_floor`. Additionally, to be consistent with the LuaJIT runtime, the branches should be flipped (`num >= 0`).